restore-sql-server-transaction-logs/Function/env.yml

```diff
@@ -0,0 +1,7 @@
+INSTANCE_ID: ""
```

Comment:
Why do you need files in different formats (JSON and YAML)?

Reply:
PS does not have yaml support out of the box. I removed the need of the setting file now. What's left is a log file.

Reply:
SGTM.